ml3d/datasets/kitti.py

```diff
@@ -75,38 +76,87 @@ def read_lidar(path):
         return np.fromfile(path, dtype=np.float32).reshape(-1, 4)
 
     @staticmethod
-    def read_label(path):
+    def read_label(path, calib):
         if not Path(path).exists():
             return None
 
         with open(path, 'r') as f:
             lines = f.readlines()
-        objects = [Object3d(line) for line in lines]
+
+        objects = []
+        for line in lines:
+            label = line.strip().split(' ')
+
+            center = np.array(
+                [float(label[11]),
+                 float(label[12]),
+                 float(label[13])]).reshape(-1, 3)
```

Comment:
is 2D tensor needed? wouldn't a 1D tensor be enough?

Reply:
Done.